docs/cli-existing-project.md

```diff
@@ -177,7 +177,6 @@ SPHINX_API_KEY=<your_api_key>
 Also, if you haven't added your node provider API key as an environment variable, please do so now. For example:
 ```
 ALCHEMY_API_KEY=<your_api_key>
-INFURA_API_KEY=<your_api_key>
```

Comment:
I know this is just an example, but including infura here is confusing because it isn't actually used in the following RPC URL example.